### DISCUSSION TITLE
broken link check

### DISCUSSION THREAD

**Discussion by JuanPabloDiaz** [cosmoart/Freesets#17] [General]
I used a [brokenlinkcheck](https://www.brokenlinkcheck.com/broken-links.php#status) and here is what I found:

![freesets-brokenlinkcheck](https://github.com/cosmoart/Freesets/assets/25883220/73aae4da-40ec-404c-bb4d-f4260368bc67)

https://remixicon.com/license
https://coreui.io/license/

Not sure why this other links are even showing since they don't exist. Do you know?

https://freesets.vercel.app/icons?page=0
https://freesets.vercel.app/ilustrations?page=0
https://freesets.vercel.app/components?page=0
https://freesets.vercel.app/icons?page=3
https://freesets.vercel.app/Free%20for%20non-commercial%20use%20with%20attribution
https://freesets.vercel.app/ilustrations?page=3

**Comment by cosmoart** [comment]
https://remixicon.com/license: It works for me.
https://coreui.io/license/: Solved.
https://freesets.vercel.app/Free%20for%20non-commercial%20use%20with%20attribution: Solved.

The other links refer to the back and forward anchors of the pagination component: if you are on page 1 the previous link will be 0 and if you are on page 2 the forward link will be 3, both pages do not exist and although the anchor cannot be clicked the anchor still exists. I have already solved.

![image](https://github.com/cosmoart/Freesets/assets/81999801/e0496d76-42da-4c1a-ba98-fab4919a9089)

**Reply by JuanPabloDiaz** [reply]
Got it. Thank for explaining.
